src/c++/perf_analyzer/model_parser.cc

```diff
@@ -265,6 +265,32 @@ ModelParser::InitTFServe(
   return cb::Error::Success;
 }
 
+cb::Error
+ModelParser::InitOpenAI(
+    const std::string& model_name, const std::string& model_version,
+    const int32_t batch_size)
+{
+  // OpenAI does not return model metadata hence we can not obtain any
+  // parameters.
+  model_name_ = model_name;
+  model_version_ = model_version;
+  max_batch_size_ = batch_size;
+
+  // OpenAI will take a single json input with a fully formed payload
+  auto in_it = inputs_->emplace("payload", ModelTensor()).first;
+  in_it->second.name_ = "payload";
+  in_it->second.datatype_ = "JSON";
+  in_it->second.shape_.push_back(1);
+
+  // OpenAI will reply with a single json output
+  auto out_it = outputs_->emplace("response", ModelTensor()).first;
+  out_it->second.name_ = "response";
```

Comment:
Who/where is expected to know this name?

Reply:
No one needs to know it, but it needed a name. Your output code would be the only place that might need it, but it should always just loop through all outputs anyways